README.md

```diff
@@ -255,6 +259,12 @@ Supported domains: BLAS, LAPACK, RNG, DFT
             <td align="center">Dynamic, Static</td>
             <td align="center">DPC++</td>
         </tr>
+        <tr>
+            <td align="center">AMD GPU</td>
+            <td align="center">AMD rocFFT</td>
+            <td align="center">Dynamic, Static</td>
+            <td align="center">DPC++</td>
+        </tr>
```

Comment:
I think we also need an entry for rocFFT in the table of [product and version information](https://github.com/FMarno/oneMKL/tree/rocfft_backend#product-and-version-information) below.